sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_client.py

```diff
@@ -251,8 +251,8 @@ def get_queue_sender(self, queue_name, **kwargs):
 
         :param str queue_name: The path of specific Service Bus Queue the client connects to.
         :rtype: ~azure.servicebus.ServiceBusSender
-        :keyword str client_identifier: A string based identifier to uniquely identify the sender instance. 
-         Service Bus will associate it with some error messages for easier correlation of errors. 
+        :keyword str client_identifier: A string based identifier to uniquely identify the sender instance.
```

Comment:
```suggestion
        :keyword str client_identifier: A string-based identifier to uniquely identify the sender instance.
```

pedantic formatting nit.   😄